src/pages/EP_ManageEPTemplate.page

```diff
@@ -0,0 +1,111 @@
+<apex:page standardController="Engagement_Plan_Template__c" extensions="EP_ManageEPTemplate_CTRL" standardStylesheets="false" showHeader="true">
+    <apex:stylesheet value="{!URLFOR($Resource.sfdo203, '/assets/styles/salesforce-lightning-design-system-vf.min.css')}" />
+    <apex:stylesheet value="{!URLFOR($Resource.CumulusStaticResources, '/npsp-slds/npsp-common.css')}" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <apex:form id="theForm">
+        <div class="sfdo203">
+            <div class="slds-spinner_container" style="z-index:999;position:fixed;display:none;" id="divLoading">
+                <div class="slds-spinner--brand slds-spinner slds-spinner--medium" style="position:fixed;top:30%;" aria-hidden="false" role="alert">
+                    <div class="slds-spinner__dot-a"></div>
+                    <div class="slds-spinner__dot-b"></div>
+                </div>
+            </div>
+            <!-- PAGE HEADER -->
+            <div class="slds-page-header" role="banner">
+              <div class="slds-grid">
+                <div class="slds-col">
+                  <p class="slds-text-heading--medium">Manage Engagement Plan Template</p>
+                  <!--h1 class="slds-text-heading- -medium">My Expenses</h1-->
+                </div>
+                <div class="slds-col slds-no-flex slds-align-middle">
+                    <div class="slds-button-group" role="group">
+                        <apex:commandButton id="saveCloseBTN" styleClass="slds-button slds-button--brand" action="{!saveClose}" value="{!$Label.stgBtnSave}" immediate="false"/>
+                        <apex:commandButton id="cancelBTN" styleClass="slds-button slds-button--neutral" action="{!cancel}" value="{!$Label.stgBtnCancel}"  immediate="true"/>
+                    </div>
+                </div>
+              </div>
+            </div>
+            <!-- / PAGE HEADER -->
+            
+            <apex:outputPanel id="panelErrors">
+                <c:UTIL_PageMessages html-class="slds-grid slds-grid--align-center slds-m-bottom--small"/>
```

Comment:
since this component does the slds styling, why did you have to add this html-class list?  I know my pages that use c:UTIL_PageMessages have never done this.